src/main/java/org/fundacion/jala/converter/view/converter/ImageConverterInterface.java

```diff
@@ -0,0 +1,154 @@
+        convertImage.setAlignmentX(LEFT_ALIGNMENT);
+        convertImage.setFont(new Font("Barlow", Font.PLAIN, FONT_SIZE));
+        convertImage.addActionListener(this::actionPerformed);
+        file = new SelectFile();
+        file.setAlignmentX(LEFT_ALIGNMENT);
+        imageSelect = new ConvertTypeSelectImage();
+        imageSelect.setAlignmentX(LEFT_ALIGNMENT);
+        setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
+        setBorder(new EmptyBorder(TOP_BORDER, LEFT_BORDER, BOTTOM_BORDER, RIGHT_BORDER));
+        settings = new OutputSettingsImage();
+        settings.setAlignmentX(LEFT_ALIGNMENT);
+        ImageIcon icon = new ImageIcon("img/loading.gif");
+        label = new JLabel();
+        label.setIcon(new ImageIcon(icon.getImage()));
+        label.setVisible(false);
+        JPanel btnContainer = new JPanel();
+        btnContainer.setLayout(new FlowLayout(FlowLayout.CENTER));
+        btnContainer.add(convertImage);
+        JPanel container = new JPanel();
+        container.setBorder(new EmptyBorder(MARGIN_SPACE, 0, MARGIN_BOTTOM_BTN_CONTAINER, 0));
+        container.setLayout(new BoxLayout(container, BoxLayout.Y_AXIS));
+        container.add(file);
+        container.add(imageSelect);
+        container.add(settings);
+        setLayout(new BorderLayout());
+        setBorder(new EmptyBorder(0, MARGIN_SPACE, MARGIN_BOTTOM_MAIN_CONTAINER, MARGIN_SPACE));
+        add(imageTitle, BorderLayout.NORTH);
+        add(container, BorderLayout.CENTER);
+        add(btnContainer, BorderLayout.SOUTH);
+    }
+
+    /**
+     * Invoked when an action occurs.
+     *
+     * @param e the event to be processed
+     */
+    @Override
+    public void actionPerformed(final ActionEvent e) {
+        LOGGER.info("start");
+        if (checkFileSelect(file.getOriginFilePath())) {
+            label.setVisible(true);
+            try {
+                LOGGER.info("Execute Try");
+                checksumLocal = getFileChecksum(file.getOriginFilePath());
+                int option = JOptionPane.showConfirmDialog(this, "File Path: "
+                        + file.getOriginFilePath() + "\nConvert to: " + imageSelect.getConvertTo()
+                        + "\nWidth size: " + settings.getWidthSize() + "\nGray scale: " + settings.isGrayScale()
+                        + "\nChecksum: " + checksumLocal, "Message confirm", JOptionPane.YES_NO_OPTION);
+                if (option == 0) {
+                    callRequest();
+                } else {
+                    label.setVisible(false);
+                }
+                LOGGER.info("finish");
+            } catch (IOException | ChecksumException ioException) {
+                ioException.printStackTrace();
+                LOGGER.error("Execute Exception");
+            }
+        }
+        LOGGER.info("Finish");
+    }
+
+    /**
+     * Obtains the request.
+     *
+     * @throws IOException when problems on inputs.
+     */
+    private void callRequest() throws IOException {
```

Comment:
Should throw "ConverterException" instead of "IOException"

Reply:
Thank you Jess, already done